engine/dumps/bank03.asm

```diff
@@ -2362,15 +2362,15 @@ Texte2cc:
 	text "１ぴきも　もってへんやんか！"
 	prompt
 
-Texte2dc:
-	text "それ　あずけたら"
+Texte2dc: ; failed deposit last mon
+	text "それ　あずけたら" ; telling you that you can't deposit your last mon
 	line "こまるんとちゃう？"
 	prompt
 
 Functione2f0:
 	call LoadStandardMenuHeader
 	ld hl, Tablee6da
-	call Functione6a4
+	call Functione6a4 ; has something to do with releasing mon from PC
```

Comment:
Don't be afraid to give something an incorrect name. Even an incorrect name will give more context, allowing it to be fixed later down the line.

Reply:
That said, if you don't feel comfortable naming the function just yet, the comment should be placed where the function is, rather than where it's used. Don't copy-paste comments.